md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/BatchAPI.java

```diff
@@ -135,11 +130,10 @@ RestWrapper list(@RequestParam(value = "page", defaultValue = "0") int startPage
                 batches.add(returnBatch);
 
             }
-            //List<Batch> batches = s.selectList("call_procedures.GetBatches", batch);
-
             restWrapper = new RestWrapper(batches, RestWrapper.OK);
             LOGGER.info("All records listed from Batch by User:" + principal.getName());
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)